Config search regression introduced in 2.3.0

I have a monorepo project (inside project multiple sub-project each with own package.json). ESLint is set up in the project root directory, and not in sub-directories with own package.json. With [this change](https://github.com/TheSavior/ESLint-Formatter/commit/888f11908b7d49d243f8960f297ea72a19a9a5db) introduced in 2.3.0 ESLint-Formatter looking in the closest package.json, but if there is no ESLint, it won't look into project root, and go straight to globally installed ESLint.

```
Using node.js path on 'osx': /usr/local/bin/node
Using eslint path on 'osx': /usr/local/bin/eslint
Unexpected error(<class 'Exception'>): Error: module.js:471
    throw err;
    ^

Error: Cannot find module '/usr/local/bin/eslint'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3

error: Error: module.js:471
    throw err;
    ^

Error: Cannot find module '/usr/local/bin/eslint'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
```

--- a/ESLint-Formatter.py
+++ b/ESLint-Formatter.py
@@ -200,7 +200,7 @@ class PluginUtils:
       normdn = PluginUtils.normalize_path(project_path)
     else:
       normdn = PluginUtils.normalize_path(dirname)
-      
+
     for d in PluginUtils.walk_up(normdn):
       matches = glob.glob(os.path.join(d, pattern))
       if matches:

--- a/ESLint-Formatter.py
+++ b/ESLint-Formatter.py
@@ -200,8 +200,7 @@ class PluginUtils:
       normdn = PluginUtils.normalize_path(project_path)
     else:
       normdn = PluginUtils.normalize_path(dirname)
-
-    
+      
     for d in PluginUtils.walk_up(normdn):
       matches = glob.glob(os.path.join(d, pattern))
       if matches:

--- a/ESLint-Formatter.py
+++ b/ESLint-Formatter.py
@@ -201,6 +201,7 @@ class PluginUtils:
     else:
       normdn = PluginUtils.normalize_path(dirname)
 
+    
     for d in PluginUtils.walk_up(normdn):
       matches = glob.glob(os.path.join(d, pattern))
       if matches:
@@ -210,13 +211,14 @@ class PluginUtils:
 
   @staticmethod
   def get_local_eslint(dirname):
-    pkg = PluginUtils.findup('package.json', dirname)
+    pkg = PluginUtils.findup('node_modules/eslint', dirname)
     if pkg == None:
       return None
     else:
       path = PluginUtils.get_pref("local_eslint_path").get(sublime.platform())
-      d = os.path.dirname(pkg)
+      d = os.path.dirname(os.path.dirname(pkg))
       esl = os.path.join(d, path)
+
       if os.path.isfile(esl):
         return esl
       else:

